pkgs/top-level/python-packages.nix

```diff
@@ -13986,6 +13999,10 @@ self: super: with self; {
       if pkgs.config.cudaSupport
       then pkgs.magma-cuda-static
       else pkgs.magma;
+    openai-triton =
+      if pkgs.config.cudaSupport
+      then self.openai-triton-cuda
+      else self.openai-triton;
```

Comment:
Isn't this a no-op? If the global `config.cudaSupport` is true, then `openai-triton` should equal `openai-triton-cuda`?

Reply:
Also maybe don't take heed of the magma snippet right above, it probably needs to be moved into the expression itself, so as to make overlays less fragile

Reply:
Yes you're right. My bad.